src/openssl/meson.build

```diff
@@ -0,0 +1,39 @@
+dep_openssl = dependency('openssl11', required: false, static: static)
+if not dep_openssl.found()
+  dep_openssl = dependency('openssl', static: static)
```

Comment:
Should this have a versioned requirement to ensure it only detects openssl 1.1.x?

Reply:
Good point, it should as we already dropped support for older versions.

Reply:
(I am honestly not sure why any distros are patching out that pkg-config file to have the wrong name.)